source/Meadow.Core/Configuration/MeadowUpdateSettings.cs

```diff
@@ -10,7 +10,7 @@ internal class MeadowUpdateSettings : IUpdateSettings
     public string AuthServer { get; set; } = "https://www.meadowcloud.co";
     public int AuthPort { get; set; } = 443;
     public string Organization { get; set; } = "Default organization";
-    public string RootTopic { get; set; } = "ota;ota/{ID}/updates";
+    public string RootTopic { get; set; } = "ota;ota/{ID}/updates;{OID}/ota/{ID}";
```

Comment:
Let's remove `ota;ota/{ID}/updates;`. When we move over to EMQX the device won't be able to connect to these anyway.

Reply:
I can do that. My only concern is that once this is merged in, then we are potentially committed to rolling out Meadow.Cloud on the next release of Meadow.Core. If there's a hotfix that's required (and that hotfix is based on `develop`), then OtA breaks until the changes to Meadow.Cloud are in production.

Reply:
I'm going to leave the existing RootTopics in until we have this tested further.